### ISSUE TITLE
Build fails: `undefined: time.Until`

### ISSUE THREAD

**Issue by shark0der** [jmhodges/howsmyssl#167]
Fresh clone from github 28adbfe:
```
howsmyssl@host:~/go/src/github.com/jmhodges/howsmyssl$ go build
# github.com/jmhodges/howsmyssl/tls18
tls18/tls.go:105: undefined: time.Until
```

Not sure what can I do.

**Comment by shark0der** [comment]
I was able to go back until 0fd73c0 and was able to build, therefore 5056cf3 is the commit which introduced the error.

**Comment by jmhodges** [comment]
time.Until was added in Go 1.8 which howsmyssl requires. (The README mentions that requirement)

**Comment by jmhodges** [comment]
Btw you can google up functions and methods that are mentioned in `undefined` errors and have pretty good luck at finding their godoc.

**Comment by shark0der** [comment]
Thanks. Apparently I'm using `go version go1.6.2 linux/amd64` which is shipped by default in ubuntu 16.04. I'll try my luck with a PPA.

Leaving this here for future reference: https://github.com/golang/go/wiki/Ubuntu
